### PR TITLE
Add Flyte as CloudEvents adopter

### PR DESCRIPTION
**Pull request by fg91**

### CODE REVIEW

**Comment by deissnerk** [comment]
@fg91 Thanks for this PR. I could merge it as is, but I wonder, if you might want to check how the logo looks on the deploy preview. Due to its shape it gets really small. Please let me know if you want to change the logo or keep it as is.

File: data/en/integrations.yaml [hunk 1]
```diff
@@ -58,6 +58,11 @@
     Eventarc is a managed infrastructure that allows you to build event-driven architectures with loosely coupled services that react to state changes.
   url: https://cloud.google.com/eventarc/docs/
   logo: googlecloud.png
+- name: Flyte
```
Comment (duglin) [inline data/en/integrations.yaml:61]: Can you move this up to before "Golioth" so the list is alphabetized?

Comment (fg91) [reply data/en/integrations.yaml:61]: Will do :+1:

Comment (fg91) [reply data/en/integrations.yaml]: https://github.com/cloudevents/cloudevents-web/pull/78/commits/1cd2f1fae70fa1fa6aea18237d1d1f703f72163b

**Comment by fg91** [comment]
> @fg91 Thanks for this PR. I could merge it as is, but I wonder, if you might want to check how the logo looks on the deploy preview. Due to its shape it gets really small. Please let me know if you want to change the logo or keep it as is.

Thanks for the hint. I replaced the image: https://github.com/cloudevents/cloudevents-web/pull/78/commits/5e63a37d28c77fa6fa4e137293f58f2faa07ee5e

**Pull request merged**